docs/topics/kapt.md

```diff
@@ -266,3 +291,20 @@ fun encodeList(options: Map<String, String>): String {
 }
 ```
 
+## Making Java's annotation processors working
```

Comment:
It is better to rephrase this part.
AP always works. Kapt simply allows to run them on all sources java+kotlin.

So I would go for something like this:

## Keep javac annotation processors

By default, kapt assumes that all annotation processors work through it and disables Java's compiler's annotation processors.  Some AP should be run by javac (for example lombok). To enable javac annotation processing use flag keepJavacAnnotationProcessors, so kapt won't disable javac annotation processing.